Config/SEGGER_RTT_Conf.h

```diff
@@ -158,9 +158,16 @@ Revision: $Rev: 21386 $
 #if ((defined(__SES_ARM) || defined(__SES_RISCV) || defined(__CROSSWORKS_ARM) || defined(__GNUC__) || defined(__clang__)) && !defined (__CC_ARM) && !defined(WIN32))
   #ifdef __ZEPHYR__

```

Comment:
does not seem against latest commit... https://github.com/zephyrproject-rtos/segger/commit/7d78f7121dc4c59396c12e60e34f3617bac5efcb

Reply:
updated.